connector_prestashop_catalog_manager/models/product_product/exporter.py

```diff
@@ -151,13 +151,14 @@ def _unit_price_impact(self, record):
         else:
             extra_to_export = record.impact_price
         tax = record.taxes_id[:1]
+        # 6 is the rounding precision used by PrestaShop for the
+        # tax excluded price.
         if tax.price_include and tax.amount_type == "percent":
-            # 6 is the rounding precision used by PrestaShop for the
-            # tax excluded price.  we can get back a 2 digits tax included
-            # price from the 6 digits rounded value
+            # we can get back a 2 digits tax included price
+            # from the 6 digits rounded value
             return {"price": round(extra_to_export / self._get_factor_tax(tax), 6)}
         else:
-            return {"price": extra_to_export}
+            return {"price": round(extra_to_export, 6)}
```

Comment:
Shouldn't we use a `float_round`?

Reply:
as in place above (and every other place in this connector) it simply uses round, I think it should work as expected with it.